python/ambassador/envoy/v3/v3httpfilter.py

```diff
@@ -268,7 +268,7 @@ def V3HTTPFilter_authv1(auth: IRAuth, v3config: 'V3Config'):
         }
 
     if auth.proto == "grpc":
-        protocol_version = auth.get('protocol_version', 'v2')
+        protocol_version = auth.get('protocol_version', "v3")
```

Comment:
Likewise.

Reply:
Wait, this isn't even the right syntax anymore, is it? `auth` is an `IRAuth` instance, and you changed `irauth.py` to have `protocol_version` be ` auth.class_member` rather than a `auth["dict_member"]`.